src/components/contentful/contentfulExternalForm/contentfulExternalForm.scss

```diff
@@ -0,0 +1,18 @@
+.ExternalForm {
+  text-align: center;
+  margin-bottom: 1rem;
+
+  &__link,
+  &__link:visited {
+    display: inline-block;
+    margin: 0 auto;
+    padding: 1rem;
+    font-size: 2rem;
+    border-radius: 5px;
+    background: #a6e8e3;
+    text-decoration: none;
+    box-shadow: -2px 3px 3px rgba(0, 0, 0, 0.6);
+    color: #000;
+    max-width: 30rem;
+  }
```

Comment:
I'm a bit against using button-styles for things that are actually links (as they lead somewhere, not trigger an action on the site). It can be confusing, as for example the sighted keyboard users try to trigger that link with spacebar, and then the site just scrolls down and doesn't activate, as it would be with an actual button. 

So maybe this is something you could also discuss with Mari? 😊 I'd suggest that let's use these styles for now, and then when you have the time for the discussion, let's change these afterward. I would imagine this isn't the only case that conversation affects 🤔 (Can't say for sure though, I'm not that deep in the code yet 😄 )

Reply:
As for these small components, we have agreed to add some quick styling and then confirm if any modifications is needed with Mari afterwards, so I guess we could make all the necessary accessibility support when required!

It'll be great to go through this kind of small component's design with Mari after launch and get her feedback or custom designs in one go. : )